pkg/envoy/lds/connection_manager.go

```diff
@@ -23,7 +26,7 @@ const (
 	prometheusInboundVirtualHostName    = "prometheus-inbound-virtual-host"
 )
 
-func getHTTPConnectionManager(routeName string, cfg configurator.Configurator, headers map[string]string) *xds_hcm.HttpConnectionManager {
+func getHTTPConnectionManager(routeName string, cfg configurator.Configurator, headers map[string]string, outbound bool) *xds_hcm.HttpConnectionManager {
```

Comment:
For some reason I don't see where the `outbound bool` is used.  Is it used? OR could we remove this?

Reply:
+1
the `outbound` bool is confusing in the method signature. At the very least, the variable should be renamed.

Reply:
had no intention to merge this like this.
I wanted  way to discuss first if we want this segmented by listener direction, just needed a way to test it for outbound to start with.